vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/split_test_frequency_cap.cc

```diff
@@ -0,0 +1,71 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/frequency_capping/exclusion_rules/split_test_frequency_cap.h"
+
+#include "base/metrics/field_trial.h"
+#include "base/optional.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ads/internal/bundle/creative_ad_info.h"
+
+namespace ads {
+
+namespace {
+
+const char kStudyName[] = "AdvertiserSplitTestStudy";
```

Comment:
i suggest starting a wiki page documenting what each of these experiments is for

Reply:
@diracdeltas I'll create one together with the blog post and will reference from there.